src/reduxSetup/reducers/rootReducer.js

```diff
@@ -1,6 +1,7 @@
 import { combineReducers } from "redux";
-import { popularMenuReducer } from "./popularMenuReducer";
+import { popularMenuReducer, specialMenuReducer } from "./popularMenuReducer";
 const RootReducer = combineReducers ({
-    menu: popularMenuReducer
+    menu: popularMenuReducer,
```

Comment:
Consider combining popularMenuReducer and specialMenuReducer into one and have only
```
const RootReducer = combineReducers ({
    menu: menuReducer,
})
```